apps/server/src/modules/h5p-editor/controller/dto/ajax/post.body.params.transform-pipe.spec.ts

```diff
@@ -77,4 +77,10 @@ describe('transform', () => {
 		const result = await ajaxBodyTransformPipe.transform(emptyAjaxPostBodyParams4);
 		expect(result).toBeUndefined();
 	});
+	it('when value contains no valid data', async () => {
```

Comment:
Whole testSuite does not follow our rules for it and describe (describe-when => it-should).
But you can not cleanup all that...
But you could add an empty line above the new test :)